src/adapters/PeopleAdapter.js

```diff
@@ -7,14 +7,33 @@ import {throwError} from 'rxjs';
  */
 export default class PeopleAdapter {
   /**
-   * Returns an observable that emits person data
+   * The status a person can have.
+   * @typedef {null|'active'|'bot'|'call'|'dnd'|'group'|'inactive'|'meeting'|'ooo'|'presenting'|'self'|'typing'} PersonStatus                      a              a      a    a
```

Comment:
There are a bunch of random characters at the end of this line.

```suggestion
   * @typedef {null|'active'|'bot'|'call'|'dnd'|'group'|'inactive'|'meeting'|'ooo'|'presenting'|'self'|'typing'} PersonStatus
```

Reply:
Wow. I didn't think I committed that, was testing the width limits 🤦‍♂